### ISSUE TITLE
How can we compile a Windows build?

### ISSUE THREAD

**Issue by themodfather360** [johang/vlc-bittorrent#41]
How can we build this for Windows?

**Comment by johang** [comment]
I don't know. I don't use Windows very much.

**Comment by themodfather360** [comment]
Fair enough. I tried it myself and wasn't able to figure it out. If anyone else knows how to compile a plugin for Windows, please let us know.

**Comment by petterreinholdtsen** [comment]
Someone is working on it in #53.  Perhaps you can help make it happen?

**Comment by evo-i** [comment]
https://github.com/johang/vlc-bittorrent/pull/53#issuecomment-2298717803

**Comment by omgbox** [comment]
https://github.com/omgbox/vlc-bittorrent

Windows Version working latest vlc 3.0.23 , 2 days of bug tracking and going with static builds.
